oracle/oracles/pd.go

```diff
@@ -231,6 +237,10 @@ func (o *pdOracle) updateTS(ctx context.Context, interval time.Duration) {
 				o.setLastTS(ts, txnScope)
 				return true
 			})
+			if o.updateInterval != currentInterval {
+				currentInterval = o.updateInterval
```

Comment:
There seems to be data race as the `updateInterval` is written in another thread. Perhaps atomic types need to be used for it.